clab/config/utils.go

```diff
@@ -13,16 +14,16 @@ import (
 )
 
 const (
-	vkNodes    = "nodes" // reserved, used for all nodes
-	vkNodeName = "node"  // reserved, used for the node's ShortName
-	vkLinks    = "links" // reserved, used for all link in a node
-	vkRole     = "role"  // optional, will default to the node's Kind. Used to select the template
-	vkFarEnd   = "far"   // reserved, used for far-end link & node info
-
-	vkSystemIP = "systemip" // optional, system IP if present could be used to calc link IPs
-	vkLinkIP   = "ip"       // optional, link IP
-	vkLinkName = "name"     // optional, from ShortNames
-	vkLinkNr   = "linknr"   // optional, link number in case you have multiple, used to calculate the name
+	vkNodeName = "node"       // reserved, used for the node's ShortName
+	vkNodes    = "clab_nodes" // reserved, used for all nodes
+	vkLinks    = "clab_links" // reserved, used for all link in a node
+	vkFarEnd   = "clab_far"   // reserved, used for far-end link & node info
+	vkRole     = "clab_role"  // optional, will default to the node's Kind. Used to select the template
+
+	vkSystemIP = "system_ip" // optional, system IP if present could be used to calc link IPs
+	vkLinkIP   = "ip"        // optional, link IP
+	vkLinkName = "name"      // optional, from ShortNames
+	vkLinkNo   = "link_no"   // optional, link number in case you have multiple, used to calculate the name
```

Comment:
same thing, let's add `clab_` prefix to indicate the significance of those vars?

and maybe then

* `clab_link_ip`
* `clab_link_name`
* `clab_link_num`
?